internal/validate/validate.go

```diff
@@ -2,26 +2,21 @@ package validate
 
 import (
 	"errors"
+	"fmt"
 	"net/http"
 	"strconv"
 	"unicode"
 )
 
-var (
-	ErrInvalidChar            = errors.New("invalid character in path")
-	ErrContentLengthMissing   = errors.New("missing Content-Length")
-	ErrContentLengthIncorrect = errors.New("incorrect Content-Length")
-)
-
 // ShortenRequest validates http request for shorten service
 func ShortenRequest(req *http.Request) (size int, err error) {
 	cl := req.Header.Get("Content-Length")
 	if cl == "" {
-		err = ErrContentLengthMissing
+		err = errors.New("missing Content-Length")
 		return
 	}
 	if size, err = strconv.Atoi(cl); err != nil {
-		err = errors.Join(ErrContentLengthIncorrect, err)
+		err = errors.Join(errors.New("incorrect Content-Length"), err)
```

Comment:
Более лаконичный способ обернуть ошибку:

```suggestion
		err = fmt.Errorf("incorrect Content-Length: %w", err)
```

Reply:
Согласен, исправил